src/backend/marsha/core/models/video.py

```diff
@@ -133,6 +135,13 @@ class Video(BaseFile):
         verbose_name=_("is video public"),
         help_text=_("Is the video publicly accessible?"),
     )
+    join_mode = models.CharField(
+        max_length=20,
+        verbose_name=_("Join discussion mode"),
+        help_text=_("Join discussion mode."),
+        choices=JOIN_MODE_CHOICES,
+        default=OPEN,
+    )
```

Comment:
Shouldn't we organise a bit the video object, for instance:
- group all settings for the video together
- or group join_mode with people asking to join and people on stage